core/src/initializers/codeConfig.ts

```diff
@@ -11,30 +14,57 @@ declare module "actionhero" {
   }
 }
 
-export class CodeConfig extends CLSInitializer {
+export class CodeConfig extends Initializer {
   constructor() {
     super();
     this.name = "codeConfig";
     this.loadPriority = 10000;
     this.startPriority = 10;
   }
 
-  async initializeWithinTransaction() {
+  async initialize() {
     api.codeConfig = {
       allowLockedModelChanges: true,
     };
   }
 
-  async startWithinTransaction() {
-    const configDir = await getConfigDir(
-      process.env.GROUPAROO_RUN_MODE === "cli:config"
-    );
-
-    const { errors } = await loadConfigDirectory(configDir);
+  async start() {
+    await CLS.wrap(async () => {
+      const configDir = await getConfigDir(
+        process.env.GROUPAROO_RUN_MODE === "cli:config"
+      );
+      const { errors } = await loadConfigDirectory(configDir);
+      if (errors.length > 0) throw new Error("code config error");
+    });
 
-    if (errors.length > 0) throw new Error("code config error");
     api.codeConfig.allowLockedModelChanges = false; // after this point in the Actionhero boot lifecycle, locked models cannot be changed
+    await loadSampleProfiles();
```

Comment:
The story called for this to **not** be awaited and run in the "background".  However, I'm concerned about creating parallel queries when using SQLite (which ui-config is guaranteed to be using).  

At the very lease, we now are using a "regular" Initializer, not CLSInitializer so failures are OK and the transaction won't be polluted by an import failure here.

Reply:
That's a good point but I'm also worried it might take a while to launch.
At a minimum, let's log per we do that so it's clear what is happening on the console.

Reply:
<img width="1150" alt="Screen Shot 2021-12-02 at 6 41 08 PM" src="https://user-images.githubusercontent.com/303226/144535695-d88071fe-a9b8-44ff-bac3-eb2356150aaa.png">

added in f89ddaa66